cloud/scope/object_storage_bucket.go

```diff
@@ -17,6 +17,18 @@ import (
 	infrav1alpha1 "github.com/linode/cluster-api-provider-linode/api/v1alpha1"
 )
 
+const bucketDataSecret = `kind: Secret
+apiVersion: v1
+metadata:
+  name: etcd-backup
+  namespace: kube-system
+stringData:
+  bucket_name: %s
```

Comment:
How about `bucket_label` versus `bucket_name`?

Reply:
I went with `bucket_name` to be consistent with the variable that is going fill in this value `s.Bucket.Name`

Reply:
Ah, makes sense.